src/app/components/nav-bar/nav-bar.component.html

```diff
@@ -8,6 +8,7 @@
       </button>
       <div ngbDropdownMenu>
         <button class="nav-bar__menu-item" (click)="navigate('/')" ngbDropdownItem>Search</button>
+        <button class="nav-bar__menu-item" (click)="navigate('/initial-setup-info')" ngbDropdownItem>Initial Setup</button>
```

Comment:
remove here, but maybe add an option to the preferences to start the setup process

Reply:
extra menu in the preferences?

Reply:
just as a routeLink in the preferences.component.ts

Reply:
done